defaults/main.yml

```
@@ -37,7 +37,11 @@ miniflux_oauth2_user_creation: ""
 miniflux_oauth2_oidc_discovery_endpoint: ""
 miniflux_pocket_consumer_key: ""
 miniflux_proxy_images: "http-only"
+# other options?
+miniflux_conf_extra_options: []
 
 miniflux_database_url: "user={{ miniflux_dbuser }} password={{ miniflux_dbpassword }} host={{ miniflux_dbhost }} dbname={{ miniflux_dbname }} sslmode=disable"
 
+backupdir: /var/_backup
```

I think we don't need the underscore
```suggestion
backupdir: /var/backup
```

---

two reasons for /var/_backup and not /var/backup
* not mix with debian /var/backups (alternatives, apt, dpkg... backups)
* separate service/usage with underscore prefix similar to to c++ programming private variables and openbsd unprivileged daemon/service users. Debian partially does (_apt, _rpc... vs Debian-exim)
https://www.reddit.com/r/learnjavascript/comments/x8n0hh/why_some_people_use_underscore_before_variable/
https://unix.stackexchange.com/questions/305124/why-do-some-usernames-on-freebsd-start-with-an-underscore
https://github.com/geerlingguy/ansible-role-ruby/issues/64: has changed, now guide ack underscore but says no special status: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#creating-valid-variable-names

what do you prefer?